docs/Windows_Installation_Instructions.md

```diff
@@ -0,0 +1,176 @@
+
+This will not only install Git, but will also provide Git BASH, a shell command for working with Git in Windows.
+
+## 3. Clone the MMS 4 Repository
+> If you already cloned the repository, make sure you are on the correct branch/tag by following the checkout step (4).
+
+Clone the repository and checkout the latest tagged version, as of the writing of these instructions.
+
+1. Open GitBash
+
+1. Navigate to the folder you want to clone the repo on, e.g., `C:\repos\mms`, but you may need to clone in your user folder if you do not have permission to create a new folder at the root drive level, ***for example***:
+
+    ```bash
+    cd /c/repos/mms
+    ```
+    
+    > If you have to use a deeply nested folder, make sure you check out the section below on how to extend the Windows maximum path length.
+
+1. Clone the `mms` repo
+
+    ```bash
+    git clone git@github.com:Open-MBEE/mms.git
+    ```
+
+1. Checkout the right branch/tag:
+
+    ```bash
+    git checkout 4.0.0-a2
+    ```
+
+## 4. Get setup
+> IMPORTANT: Windows PowerShell is the recommended shell for the remaining steps.
+
+1. Open "Windows PowerShell".
+1. Navigate to the folder where you cloned the `mms` repository.
+1. Copy the example MMS configuration file
+   * `cp .\example\src\main\resources\application-test.properties .\example\src\main\resources\application.properties`
+   > This configuration file is intended to work with the `docker-compose` file in the MMS repo
+1. Make sure that `docker-compose` is available by running:
+
+    ```bash
+    docker-compose --version
+    ```
+    You should see something like this:
+    ```bash
+    docker-compose version 1.27.2, build 18f557f9
+    ```
+
+## 5. Start and Monitor MMS
+1. Start up the containers
+
+    ```bash
+    docker-compose up -d
+    ```
+
+    You should see something like this:
+
+    ```bash
+    Creating network "mms_default" with the default driver
+    Creating mms_postgres_1      ... done
+    Creating mms_elasticsearch_1 ... done
+    Creating mms                 ... done
+    ```
+
+    > If you are running this for the first time, you will see a much longer output.
+
+    > If you modify the `application.properties` configuration file you will have to rebuild the `MMS` image by running: `docker-compose up build`
+
+1. Make sure the containers are running:
+
+    ```bash
+    docker-compose ps
+    ```
+
+    You should see something like this:
+
+    ```bash
+           Name                      Command               State                       Ports
+    -------------------------------------------------------------------------------------------------------------
+    mms                   java -Djdk.tls.client.prot ...   Up      0.0.0.0:8080->8080/tcp
+    mms_elasticsearch_1   /usr/local/bin/docker-entr ...   Up      0.0.0.0:9200->9200/tcp, 0.0.0.0:9300->9300/tcp
+    mms_postgres_1        docker-entrypoint.sh postgres    Up      0.0.0.0:5432->5432/tcp
```

Comment:
Again, not super important, but maybe we need to add one line for `minio`:

`mms_minio_1             /usr/bin/docker-entrypoint ...   Up      0.0.0.0:9000->9000/tcp`